modules/account/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="error_sign_in_failed">Failed to sign in.</string>
+    <string name="please_enter_a_valid_email">Please enter a valid email.</string>
+    <string name="failed_to_send_email">Failed to send email.</string>
+    <string name="email_sent">Email sent.</string>
+    <string name="google_icon">Google Icon</string>
+    <string name="sign_in_with_google">Sign in with Google</string>
+    <string name="or">OR</string>
+    <string name="your_account">Your Account</string>
+    <string name="password">Password</string>
+    <string name="hide_password">Hide password</string>
+    <string name="show_password">Show password</string>
+    <string name="forgot_password">Forgot Password?</string>
+    <string name="don_t_have_an_account_yet">Don\'t have an Account yet?</string>
+    <string name="ok">OK</string>
+    <string name="cancel">Cancel</string>
+    <string name="create_a_new_account">Create a new Account</string>
+    <string name="please_fill_out_the_details_below_to_create_your_new_account">Please fill out the details below to create your new account.</string>
+    <string name="e_mail_address">E-Mail Address</string>
+    <string name="first_name">First Name</string>
+    <string name="last_name">Last Name</string>
+    <string name="gender">Gender</string>
+    <string name="select_gender">Select Gender</string>
+    <string name="date_of_birth">Date of Birth</string>
+    <string name="select_date">Select Date</string>
```

Comment:
we have removed these from the registration screen (name, gender and birthday). Should we remove the strings as well as they are not used?

Reply:
good idea, i had forgotten about the string resources